include/BDSFieldEMAcceleration.hh

```diff
@@ -0,0 +1,28 @@
+//
+// Created by Boogert Stewart on 16/08/2024.
+//
+
+#ifndef BDSIM_BDSFIELDEMACCELERATION_HH
```

Comment:
Header guards across the code are of the form BDSCLASSNAMEWITHOUTUNDERSCORE_H  (1 H)